src/collections/mapContains/MapContains1.kts

```diff
@@ -0,0 +1,21 @@
+package collections.mapContains
+
+import java.util.concurrent.ConcurrentHashMap
+
+val scores = ConcurrentHashMap<String, Int>()
+scores["Steve"] = 9
+scores["Rachel"] = 11
+
+val scoresHigherThan10 = scores.filterValues { it > 10 }
+
+scores.contains("Steve")
+scores.contains("Rachel")
+
+scoresHigherThan10.contains("Steve")
+scoresHigherThan10.contains("Rachel")
```

Comment:
Not sure if it is clear enough that the question is about the results of these calls. Should I maybe adjust it (e.g. expand the question below)?
Adding a `println` for every line might be a bit verbose.

Reply:
I think adding `println` 4 times is clear. The new lines will appear in output but I think it'll be clear that the answers replaced new lines with spaces for brevity.